src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,340 @@
+      })
+
+**Adding additional Spans to the transaction**
+
+The next example is called in the `processItem` function from the code snippet above. Our SDK can determine if there is a current open `transaction` and add all newly created spans as child operations to the `transaction`. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`.
+
+    function processItem(item, transaction) {
+      const span = transaction.child({ op: "http", description: "GET /" })
+    
+      return new Promise((resolve, reject) => {
+        http.get(`/items/${item.id}`, (response) => {
+          response.on('data', () => {});
+          response.on('end', () => {
+            span.setTag("http.status_code", response.statusCode);
+            span.setData("http.foobarsessionid", getFoobarSessionid(response));
+            span.finish();
+            resolve(response);
+          });
+        });
+      });
+    }
+
+### Node.js
+
+To use our APM features, you need to use a beta release.
+
+    $ npm install @sentry/node@5.10.0-rc.0
+    $ npm install @sentry/apm@5.10.0-rc.0
+
+**Sending Traces**
+
+To send any traces, set the `tracesSampleRate`to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+    const Sentry = require("@sentry/node");
+    const Apm = require("@sentry/apm"); // This is required since it patches functions on the hub
+    
+    Sentry.init({
+      dsn: "___PUBLIC_DSN___",
+      tracesSampleRate: 0.1
+    });
+
+**Automating Traces**
+
+It’s possible to add tracing to all popular frameworks; however, we provide pre-written handlers only for Express.js.
+
+    const Sentry = require("@sentry/node");
+    const express = require("express");
+    const app = express();
+    
+    // RequestHandler creates a separate execution-context using domains, so that every transaction/span/breadcrumb has it's own Hub to be attached to
+    app.use(Sentry.Handlers.requestHandler());
+    // TracingHandler creates a tracing for every incoming request
+    app.use(Sentry.Handlers.tracingHandler());
+    
+    // the rest of your app
+    
+    app.use(Sentry.Handlers.errorHandler());
+    app.listen(3000);
+
+The following operations have span capabilities within a transaction:
+
+- HTTP requests made with `request` or `get` calls using native `http` and `https` modules
+- Middlewares (Express.js only)
+
+To enable them:
+
+    $ npm install @sentry/integrations@5.8.0-beta.0
+
+    const Sentry = require("@sentry/node");
+    const Integrations = require("@sentry/integrations");
+    const express = require("express");
+    const app = express();
+    
+    Sentry.init({
+      dsn: "___PUBLIC_DSN___",
+      tracesSampleRate: 0.1,
+      integrations: [
+        // enable HTTP calls tracing
+        new Sentry.Integrations.Http({ tracing: true }),
+        // enable Express.js middleware tracing
+        new Integrations.Express({ app })
+      ],
+    });
```

Comment:
Wrap this in 3 backticks + javascript
But this code also needs changes, this is also a todo we need to check if this is true

```suggestion
    const Sentry = require("@sentry/node");
    const Apm = require("@sentry/apm");
    const express = require("express");
    const app = express();
    
    Sentry.init({
      dsn: "___PUBLIC_DSN___",
      tracesSampleRate: 0.1,
      integrations: [
        // enable HTTP calls tracing
        new Sentry.Integrations.Http({ tracing: true }),
        // enable Express.js middleware tracing
        new Apm.Integrations.Express({ app })
      ],
    });
```

Reply:
@HazAT Okay. Let me know when there's a new code example.